### PR TITLE
Fix all major typos

### PR DESCRIPTION
**Pull request by GameCubeGBA**
A somewhat big PR, probably should split into multiple pieces, but since they all deal with typos, I don't know how to split this up in a way that makes sense without having a separate PR for every change.

Nevertheless, this deals with primarily typos in user-facing strings; strings the user will read while using the program.

I don't expect this to be merged as-is, but if it were to be, there would be no issue since this contains all the changes I wanted to make.

### CODE REVIEW

**Comment by ryanmkurtz** [comment]
I'll scan through them in the coming days.  I think it's worth it to get typos out of the code base.

File: Ghidra/Features/Base/src/main/java/ghidra/program/util/SymbolicPropagator.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
 import ghidra.util.exception.*;
 import ghidra.util.task.TaskMonitor;
 
-public class SymbolicPropogator {
+public class SymbolicPropagator {
```
Comment (astrelsky) [inline Ghidra/Features/Base/src/main/java/ghidra/program/util/SymbolicPropagator.java:42]: This change here will *probably* get rejected. I *think* I recall seeing a pr previously that addressed this one but was deemed not worth it.

If you stick to typos in documentation and error messages, things the user sees, it should be ok.

**Comment by agatti** [comment]
I believe you missed `decendent` -> `descendent` (strings: Ghidra/Framework/FileSystem/src/main/java/ghidra/framework/store/FileSystem.java, Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/app/plugin/processors/sleigh/DecisionNode.java, Ghidra/Features/GhidraServer/src/main/java/ghidra/server/store/RepositoryFolder.java variables: Ghidra/Features/Decompiler/ghidra_scripts/classrecovery/RTTIWindowsClassRecoverer.java).

Rather than creating a new PR myself for that only for these maybe you could deal with those as well and add those changes to your PR instead? :)

File: Ghidra/Features/Base/src/main/java/ghidra/program/util/SymbolicPropagator.java (hunk 1 above)
Comment (ryanmkurtz) [reply Ghidra/Features/Base/src/main/java/ghidra/program/util/SymbolicPropagator.java:42]: Agreed, we don't want to break the API, at least not right now.  Like @astrelsky said, if you make this PR just about Strings we should be able to take it.

Comment (ryanmkurtz) [reply Ghidra/Features/Base/src/main/java/ghidra/program/util/SymbolicPropagator.java:42]: Agreed, we don't want to break the API, at least not right now. Like @astrelsky said, if you make this PR just about Strings we should be able to take it.

**Review by ryanmkurtz — CHANGES_REQUESTED** [review]

**Comment by ryanmkurtz** [comment]
We can reopen if you come back to this and leave out the changes to the API.

**Pull request closed without merging**